addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -3,8 +3,8 @@
     // Common: left semi-trophy icon for certifications
     .o_survey_kanban_card_certification {
         background-image:
-            linear-gradient(rgba(255,255,255,.9),
-                            rgba(255,255,255,.9)),
+            linear-gradient(rgba($o-view-background-color,.75),
+                            rgba($o-view-background-color,.75)),
```

Comment:
just wondering, why it's a gradient and not a solid bg color?

Reply:
This gradient is placed in front of the background image to simulate an opacity (doesn't work with a solid bg color) ... but now that we're talking about it, I think it will be better to apply the opacity directly in the svg. I don't think this svg is used anywhere else 🤔.
What do you think ?

Reply:
> I think it will be better to apply the opacity directly in the svg.

I agree but let's keep it like that for now, it's out of scope and the dark-mode PR will be huge, better to limit changes to what is strictly related to color management 👍